source/reference/json/cloud-backup-schedule-config-file.txt

```diff
@@ -0,0 +1,241 @@
+       ``exportBucketId``
+     - string
+     - Unique 24-hexadecimal character string that identifies the |aws|
+       Bucket.
+
+   * - ``export.``
+       ``frequencyType``
+     - string
+     - Human-readable label that indicates the rate at which the export
+       policy item occurs.
+
+   * - ``policies``
+     - array
+     - Rules set for this backup schedule.
+
+   * - ``policies.``
+       ``id``
+     - string
+     - Unique 24-hexadecimal digit string that identifies this backup
+       policy.
+
+   * - ``policies.``
+       ``policyItems``
+     - array
+     - List that contains the specifications for one policy.
+
+   * - ``policies.``
+       ``policyItems.``
+       ``frequencyInterval``
+     - integer
+     - Number that indicates the frequency interval for a set of
+       snapshots. A value of ``1`` specifies the first instance of the
+       corresponding ``frequencyType``.
+
+       - In a monthly policy item, ``1`` indicates that the monthly
+         snapshot occurs on the first day of the month and ``40``
+         indicates the last day of the month.
+
+       - In a weekly policy item, ``1`` indicates that the weekly
+         snapshot occurs on Monday and ``7`` indicates Sunday.
+
+       - In an hourly policy item, you can set the frequency interval to
+         ``1``, ``2``, ``4``, ``6``, ``8``, or ``12``. For hourly policy
+         items for NVMe clusters, MongoDB Cloud only accepts ``12`` as
+         the frequency interval value.
+
+   * - ``policies.``
+       ``policyItems.``
+       ``frequencyType``
+     - string
+     - Human-readable label that identifies the frequency type
+       associated with the backup policy.
+
+   * - ``policies.``
+       ``policyItems.``
+       ``retentionUnit``
+     - string
+     - Unit of time in which MongoDB Cloud measures snapshot retention.
+
+   * - ``policies.``
+       ``policyItems.``
+       ``retentionValue``
+     - integer
+     - Duration in days, weeks, or months that MongoDB Cloud retains the
+       snapshot. For less frequent policy items, MongoDB Cloud requires
+       that you specify a value greater than or equal to the value
+       specified for more frequent policy items. 
+
+       For example: If the hourly policy item specifies a retention of two
+       days, you must specify two days or greater for the retention of the
+       weekly policy item. 
+
+   * - ``referenceHourOfDay``
+     - integer
+     - Hour of day in Coordinated Universal Time (UTC) that represents
+       when MongoDB Cloud takes the snapshot.
+
+   * - ``referenceMinuteOfHour``
+     - integer
+     - Minute of the ``referenceHourOfDay`` that represents when MongoDB
+       Cloud takes the snapshot.
+
+   * - ``restoreWindowDays``
+     - integer
+     - Number of previous days that you can restore back to with
+       Continuous Cloud Backup accuracy. You must specify a positive,
+       non-zero integer. This parameter applies to continuous cloud
+       backups only.
```

Comment:
Question: Is Continuous Cloud Backup capitalized in title case or not? We have two different instances here in the same description.

Reply:
So I see a mix of it throughout our cloud docs repo and the API spec. I'm going to stick with lowercasing it since it seems to be used more often in cloud docs.